CODEOWNERS

```diff
@@ -21,7 +21,7 @@
 
 /Workbooks/AKS/** @BLuEScioN @bragi92 @microsoft/applicationinsights-devtools
 
-/Workbooks/Azure*Active*Directory*Conditional*Access/** @danielwood95 @KranthiKiranerusu @jssmomo @sabinasmith
+/Workbooks/Azure*Active*Directory*Conditional*Access/** @KranthiKiranerusu @jssmomo @microsoft/azure-ad-workbooks
 /Workbooks/Azure*Active*Directory*Domain*Services/** @MikeStephens-MikeStephens
 /Workbooks/Azure*Active*Directory*TroubleShooting/** @danielwood95 @KranthiKiranerusu
 /Workbooks/Azure*Active*Directory/** @danielwood95 @KranthiKiranerusu @sukhong
```

Comment:
you don't want to touch any of the other aad ones that have your name on them?  just this one folder?